modules/bmo-troubleshooting-latency-when-deleting-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ipi-install-troubleshooothing-latency-when-deleting-a-baremetalhost-resource_{context}"]
+
+= Troubleshooting latency when deleting a BareMetalHost resource
+
+When the Bare Metal Operator (BMO) deletes a `BareMetalHost` resource, Ironic deprovisions the bare-metal host with a process known as cleaning. If cleaning does not succeed, deleting the `BareMetalHost` resource takes considerable time and may not fully delete. The deleting process retries three times.
```

Comment:
```suggestion
When the Bare Metal Operator (BMO) deletes a `BareMetalHost` resource, Ironic deprovisions the bare-metal host with a process called cleaning. When cleaning fails, the deletion of the `BareMetalHost` resource by the BMO takes considerable time and might not succeed. The deleting process retries three times.
```
"might" instead of "may" for translatability
a little unclear to me who then is deleting things if the cleaning fails; implies that a user might go in and try manually but it takes a long time and might not work

Reply:
Actually, the problem is a little bit different. Users get impatient when it takes to long and they try to manually override too soon. So a user might go in and try manually, and create problems. So we're trying to steer them in the right direction about it taking time without overloading them with internal implementation details of what's happening. People were using a workaround found here https://bugzilla.redhat.com/show_bug.cgi?id=1837505#c15, which isn't a good idea; namely, they should disable cleaning rather than removing finalizers.